libs/expo/shared/ui-components/src/lib/Avatar/Avatar.tsx

```diff
@@ -85,7 +89,9 @@ export function Avatar(props: IAvatarProps) {
           justifyContent: 'center',
         }}
       >
-        {imageUrl ? (
+        {loading ? (
```

Comment:
this is not a triple ternary and really hard to read.

Could we split this into 

```javascript
{ !!loading && <Loading /> }

{ !loading && !imageUrl ... }

{ !loading && !!imageUrl ... }
  <Image ...

}
```

Reply:
done

Reply:
ty!